hr_holiday_notify_employee_manager/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+HR Holiday Notify Employee Manager
+==================================
+
+This module extends the functionality of the Leaves App to make possible
```

Comment:
This said, a good use case for this module would be a company with the following requirements:
* There are several followers by department.
* Not all of the followers of the department can approve the leaves, but just the ones that are explicitly managers of the employee requesting the leave.

In this case you will have to configure the system in the following way:
* Configure all the followers of the department to not receive email notifications on leaves requests.
* Check the box *Leave Requests notified to employee's manager* added by this module.